db/migrate/20180404102429_update_pay_scale_data.rb

```diff
@@ -0,0 +1,61 @@
+class UpdatePayScaleData < ActiveRecord::Migration[5.1]
```

Comment:
This looks to add labels to existing pay scales but that job seems to have been done by the previous migration `AddPayScaleData`, do we need this one?

Reply:
@tahb  It updates the existing labels.

I transferred the existing rake task in add_payscale_data and then did the update as I didnt want to just drop the first step.